data/sql/updates/pending_db_world/rev_1765116411434199400.sql

```diff
@@ -0,0 +1,5 @@
+-- SpecialFlags 0 to 1, make Blessing of Zim'Torga repeatable
+UPDATE `quest_template_addon` SET `SpecialFlags` = 1 WHERE (`ID` = 12618);
```

Comment:
```suggestion
UPDATE `quest_template_addon` SET `SpecialFlags` = `SpecialFlags` = | 1 WHERE (`ID` = 12618);
```

When using Bitmask values you always do Flags = Flags or Bitmask = Bitmask, | add or &~ remove.

Check: https://www.azerothcore.org/wiki/sql-standards flags and bits